.travis.yml

```diff
@@ -46,6 +51,6 @@ deploy:
     file_glob: true
     file: "netdata*.tar.*"
     on:
-      condition: $CC = gcc
+      condition: $CC = gcc && $TRAVIS_OS_NAME
```

Comment:
`$TRAVIS_OS_NAME` is always true isn't it? you mean `$TRAVIS_OS_NAME == "linux"` perhaps?